indico/htdocs/js/indico/RegistrationForm/registrationform.js

```diff
@@ -59,6 +59,7 @@ ndRegForm.factory('regFormFactory', function($resource, $http, editionurl, displ
     defaults.common = defaults.common || {};
     defaults.get = defaults.get || {};
     defaults.get['Content-Type'] = defaults.common['Content-Type'] = 'application/json';
+    defaults.common['X-CSRF-Token'] = $('#csrf-token').attr('content');
 
     var sectionurl = editionurl + 'sections/:sectionId';
```

Comment:
err yeah, I meant `sectionURL` etc. But unseparated words in a single name are pretty ugly